src/platform/packages/shared/kbn-scout/README.md

```diff
@@ -338,6 +338,74 @@ node scripts/scout.js start-server [--stateful|--serverless=[es|oblt|security]]
 
 This command is useful for manual testing or running tests via an IDE.
 
+#### Generating Tests with Scout Recorder
+
+Scout provides a `record` command that uses Playwright's test recorder to generate Scout-compatible tests. This is a great way to quickly create test scaffolding by recording your interactions with Kibana.
+
+**Basic Usage:**
+
+```bash
+node scripts/scout.js record --stateful --plugin apm-plugin --role admin
+```
+
+Please note that the plugin name is the name of the plugin as defined in the `package.json` file, without the `@kbn` scope.
+
+**Available Options:**
+
+- **`--stateful`** or **`--serverless=[es|oblt|security]`**: Specifies the deployment type (required)
+- **`--plugin`**: Plugin name as defined in package.json, without the @kbn scope (e.g., `maps-plugin`, `apm-plugin`) (required)
+- **`--role`**: Role to authenticate as during recording (default: `admin`)
+- **`--url`**: Starting URL path to navigate to when recorder starts (e.g., `/app/maps`). Will attempt to auto-detect from plugin.ts if not provided (optional)
+- **`--testDirectory`**: Test directory to save the test file to (default: `test/scout/ui`)
+- **`--output`**: Output test file name with extension (default: `recorded_test.spec.ts`)
+- **`--parallel`**: Generate test using `spaceTest` for parallel execution (default: `false`)
+
+**How It Works:**
+
+1. Starts Kibana and Elasticsearch servers
+2. Opens Playwright's test generator with your browser
+3. Records your interactions as you perform test actions
+4. Transforms the recorded code to Scout format with:
+   - Correct Scout package imports (`@kbn/scout`, `@kbn/scout-oblt`, or `@kbn/scout-security`)
+   - Scout fixtures (`page`, `pageObjects`, `browserAuth`)
+   - Conversion of data-test-subj attributes where detected
+   - Detection and replacement of common page object patterns
+   - Authentication handling via `browserAuth` fixture
+   - Appropriate deployment tags (`@ess`, `@svlOblt`, etc.)
+5. Saves the test file to your plugin's test directory
+6. Creates Playwright config if it doesn't exist
+
+**Example Usage:**
+
+```bash
+# Generate test for Maps plugin (stateful) - browser starts at Maps app
+node scripts/scout.js record --stateful --plugin x-pack/platform/plugins/shared/maps --url /app/maps --output full_screen_mode
+
+# Generate test for APM plugin (serverless observability) - browser starts at APM app
+node scripts/scout.js codegen --serverless=oblt --plugin x-pack/solutions/observability/plugins/apm --url /app/apm --output service_inventory
```

Comment:
We're still referencing the non-existent `codegen` command here. I think it's because we applied my suggestion using automatically but it needs to be applied to all sample commands in the code block.